### PR TITLE
Parameter fix

### PR DESCRIPTION
**Pull request by Evilcat182**
Fixed issues with -Before and -After Parameter

### CODE REVIEW

**Comment by phlcrny** [comment]
Hi @Evilcat182 

Thanks for spotting and fixing these! The logic looks good to me, are you willing/able to remove the whitespace on L98 and fix the space on the if statement on L99 (for consistency)? I don't mind doing it myself post merge but I'd prefer to not immediately alter your changes!

Whichever you prefer, I'll try and get this out as `0.4.1` ASAP.

**Review by phlcrny — APPROVED** [review]

File: CCMLogs/CCMLogs.psm1 [hunk 1]
```diff
@@ -87,14 +87,17 @@ function Read-LogEntry
...
         {
             if ($Timestamp -lt $After)
             {
                 Write-Debug -Message "Timestamp is before '$After' cut-off."
                 Continue
             }
-
+        }
+        
+        if($Before)
```
Comment (phlcrny) [inline CCMLogs/CCMLogs.psm1]: 1. Doesn't show in GitHub but in VSCode I see whitespace on L98 level with the if statement, can this be removed.
2. Can you add a space between the if on L99 and the opening bracket.

**Comment by Evilcat182** [comment]
Hi @phlcrny 
I've added a new commit with the requested formatting changes.

**Review by phlcrny — APPROVED** [review]

**Comment by phlcrny** [comment]
Thanks! Really appreciate the contribution.

**Pull request merged**